testing/src/main/java/io/stargate/it/http/GraphqlTest.java

```diff
@@ -2119,6 +2121,50 @@ public void udtsTest() {
             });
   }
 
+  @Test
+  @DisplayName("Should calculate number of orders using count aggregation")
+  public void insertOrdersAndCount() {
+    OrdersInput order1 =
+            OrdersInput.builder()
+                    .prodName("p1")
+                    .customerName("c1")
+                    .price("3199.99")
+                    .description("d1")
+                    .build();
+    OrdersInput order2 =
+            OrdersInput.builder()
+                    .prodName("p2")
+                    .customerName("c2")
+                    .price("3199.99")
+                    .description("d2")
+                    .build();
+
+    ApolloClient client = getApolloClient("/graphql/betterbotz");
+    InsertOrdersMutation order1Mutation =
+            InsertOrdersMutation.builder()
+                    .value(order1)
+                    .build();
+
+    InsertOrdersMutation order2Mutation =
+            InsertOrdersMutation.builder()
+                    .value(order2)
+                    .build();
+
+    assertThat(getObservable(client.mutate(order1Mutation)).getInsertOrders().isPresent()).isTrue();
+    assertThat(getObservable(client.mutate(order2Mutation)).getInsertOrders().isPresent()).isTrue();
+
+    // execute aggregation with count()
+    GetOrdersWithCountQuery countQuery = GetOrdersWithCountQuery
+            .builder()
+            .value(order1)
+            .aggregationField("id")
+            .build();
+
+    GetOrdersWithCountQuery.Data result = getObservable(client.query(countQuery));
```

Comment:
Currently, this test fails with:
```
INFO  [OkHttp http://127.0.2.1:8080/...] 2021-04-28 15:06:11,189 GraphqlTest.java:2278 - GraphQL error found in test: [Validation error of type FieldUndefined: Field '_int_function' in type 'Orders' is undefined @ 'Orders/values/_int_function']
```
The submitted schema is:
```
INFO  [Exec Stream Pumper] 2021-04-28 15:06:11,176 ProcessRunner.java:78 - stargate0-0> WARN  [dw-133 - POST /graphql/betterbotz] 2021-04-28 15:06:11,176 GraphQL.java:508 - Query failed to validate : 'query GetOrdersWithCount($value: OrdersInput, $aggregationField: String!) { Orders(value: $value) { __typename pageState values { __typename id prodId prodName customerName address description price sellPrice count: _int_function(name: "count", args: [$aggregationField]) } } }'
```
If I understand this correctly, we need to generate now the `_int_function` at the java-backend side, right?
However, it seems that the validation happens before it goes to our logic. @olim7t Could you point me to a place where this should be handled?

Reply:
Yes, you need to implement the new field before you can reference it in a test.